lib/system.ps1

```diff
@@ -162,7 +162,10 @@ function env($name, $global, $val) {
 
 function strip_path($orig_path, $dir) {
     Show-DeprecatedWarning $MyInvocation 'Split-PathLikeEnvVar'
-    Split-PathLikeEnvVar -Name $dir -Path $orig_path
+    if ($dir -is [string]) {
+        $dir = @($dir)
+    }
+    Split-PathLikeEnvVar -Pattern $dir -Path $orig_path
```

Comment:
```suggestion
    Split-PathLikeEnvVar -Pattern @($dir) -Path $orig_path
```

Reply:
Strongly agree. it is much simpler expression than the original one.